frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -332,5 +333,9 @@ export const dashboardLogic = kea({
                 cache.draggingToastId = null
             }
         },
+        refreshDashboardItem: async ({ id }) => {
+            const dashboardItem = await api.get(`api/dashboard_item/${id}`)
```

Comment:
I'm not 100% how this logic works, but when you load a dashboard if first loads `/api/dashboard/:int/`, which includes all the dashboard items, but then it also loads each DashboardItem individually, which noticably slows down loading on large dashboards and is redundant